neuralcompression/models/deep_video_compression.py

```diff
@@ -356,7 +356,9 @@ def __init__(
                 )
             )
             if use_gdn:
-                self.layers.append(SimplifiedInverseGDN(filter_channels))
+                self.layers.append(
+                    GeneralizedDivisiveNormalization(filter_channels, inverse=True)
```

Comment:
I think this introduces a bug. We need the inverse transformation.

Also, did you test this? I don't know if it would reproduce the results the same way as my original reproduction.